tests/py_test.py

```diff
@@ -1326,6 +1328,36 @@ def backward_compatible_run_tests(**kwargs):
 
     return True
 
+wait_for_server = threading.Lock()
+server_instance_cv = threading.Condition()
+server = 0
+_ONE_DAY_IN_SECONDS = 60 * 60 * 24
+
+class MonsterStorage(monster_test_grpc_fb.MonsterStorageServicer):
+
+  def Store(self, request, context):
+    return monster_test_grpc_fb.Stat(id='Hello, %s!' % request.name)
+
+def run_server():
+  server = grpc.server(futures.ThreadPoolExecutor(max_workers=10))
+  monster_test_grpc_fb.add_MonsterStorageServicer_to_server(MonsterStorage(), server)
+  server.add_insecure_port('[::]:50051')
+  wait_for_server.acquire()
+  server.start()
+  wait_for_server.release()
+  server_instance_cv.notify(n=1)
+  try:
+    while True:
+      time.sleep(_ONE_DAY_IN_SECONDS)
+  except KeyboardInterrupt:
```

Comment:
These tests should run unattended, i.e. it should quit automatically after 1 rpc has succesfully been sent back and forth.